src/input/playlist.rs

```diff
@@ -99,6 +104,31 @@ impl CurrentProgram {
                     self.json_mod = json.modified;
                     *self.nodes.lock().unwrap() = json.program;
 
+                    self.get_current_clip();
+                    self.index.fetch_add(1, Ordering::SeqCst);
+                }
+            }
+        } else if is_remote(&self.json_path.clone().unwrap()) {
+            let resp = reqwest::blocking::Client::new()
+                .head(self.json_path.clone().unwrap())
+                .send()
+                .unwrap();
+
+            if resp.status().is_success() {
+                let headers = resp.headers().clone();
+                let last_modified = headers.get(reqwest::header::LAST_MODIFIED).unwrap();
```

Comment:
The unwarp looks to unstable for me. Maybe it lead to crash, when something goes wrong.